### DISCUSSION TITLE
Suggest ABACUS could output the version

### DISCUSSION THREAD

**Discussion by pxlxingliang** [deepmodeling/abacus-develop#695] [Ideas]
Suggest ABACUS could output the used version in the log file 
And also run a simple command like : "abacus --version" to catch the version.

**Comment by mohanchen** [comment]
That's an excellent idea! Who will be suitable to add the new feature? @pxlxingliang @caic99

**Comment by caic99** [comment]
> That's an excellent idea! Who will be suitable to add the new feature? @pxlxingliang @caic99

Hi @mohanchen ,
Since abacus does't support any arguments, printing abacus version information in output log is a better choice for now. 
And, how to decide the version number? @dyzheng
